.circleci/config.yml

```diff
@@ -269,6 +275,26 @@ jobs:
           path: libs/SalesforceHybrid/build/outputs/androidTest-results/
       - run: *run-danger
 
+  test-RestExplorer:
+    <<: *defaults
+    environment:
+      - CURRENT_LIB: "RestExplorer"
+    steps:
+      - *attach_workspace
+      - restore_cache: *restore-gradle-cache
+      - restore_cache: *restore-ruby-cache
+      - run: *determine-tests-to-run
+      - run: *launch-emulator
+      - run: *wait-for-emulator
+      - run: *run-unit-tests
```

Comment:
Edit the run unit tests function in ci-helper script to run `:native:NativeSampleApps:RestExplorer:connectedAndroidTest` when currentLib is `RestExplorer`.